tools/monitoring/mysql/tables.sql

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+# These tables only need to be created once,
+# and they should be created before any run of the application
+
+CREATE TABLE ErrorLogs
+(
+  ID           int           NOT NULL AUTO_INCREMENT,
+  ErrorPattern varchar(4095) NOT NULL,
+  ErrorMsg     varchar(4095) NOT NULL,
+  Jobname      varchar(1023) NOT NULL, /*e.g. pull-knative-serving-integration-tests*/
+  PRNumber     int, /*pull request number; null for non pull jobs*/
+  BuildLogURL  varchar(1023) NOT NULL, /*link to build-log.txt file*/
+  TimeStamp    timestamp, /* stamps the time the record is added*/
+  PRIMARY KEY (ID)
+);
+
+CREATE TABLE Alerts
```

Comment:
Do we want a primary key for the alerts table? Maybe make the ErrorPattern the primary key so it's automatically indexed?

Reply:
yes. there should be an id here as well.

Reply:
We cannot make error pattern the primary key - it is not unique for each record

Reply:
I was suggesting to have another column ID here and make it the primary key.

Reply:
+1 on adding column id and make it the primary key. 

If the error pattern is not unique, remove `UNIQUE` from the column definition? And what makes the error pattern not unique in the alerts table?

Reply:
Sorry, I thought you are talking about the other table, because I see "PRIMARY KEY (ID)" quoted.

Yes, in the Alert table the pattern is unique and I think we should make it a primary key. In this way it is easier to index. @srinivashegde86 I am not sure if you made the same assumption as me. In this case I think what Joyce said made sense.

Reply:
I think there's a restriction on the length of the primary key, I don't remember the exact limit though. varchar(4095) *may* be too large to make it the primary key

Reply:
Let's add an auto id here then. In practice this table shouldn't have too many records and we probably don't need indexing